app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,23 @@
 class PostsController < ApplicationController
   def index; end
 
-  def show; end
+  def show
+    @post = Post.where(id: params[:id])[0]
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    author = User.find(params['post']['author_id'].to_i)
+    title = params['post']['title']
+    text = params['post']['text']
+    @post = Post.new(author: author, title: title, text: text)
+
+    respond_to do |format|
+      @post.save
+      format.html { redirect_to "#{users_path}/#{current_user.id}" }
+    end
```

Comment:
- [ ] Let's create an error handling in the case when the post is not created, let's render the `new` action for the user to create a new post. See suggestion below;

```suggestion
    respond_to do |format|
      if @post.save
      format.html { redirect_to "#{users_path}/#{current_user.id}" }
      else
      format.html { render :new }
      end
    end
```